### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by paul0vinicius**
#### What is the purpose of this pull request?

Update the documentation according to the minicart model.

#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,22 +19,42 @@ See our [LTS policy](https://github.com/vtex-apps/awesome-io#lts-policy) for mor
...
-Then, add `footer` block into our app theme, like we do in our [Dreamstore app](https://github.com/vtex-apps/dreamstore/blob/master/store/blocks.json).
+Then, add `footer` block into our app theme, like we do in our [Store theme app](https://github.com/vtex-apps/store-theme/blob/master/store/blocks.json). 
+
+Now, you can change the behavior of the footer block. See an example of how to configure: 
+```json
+"footer": {
+    "props": {
+      "socialNetworks": [{"links": [{ "url":"www.facebook.com", "title":"Facebook" }, 
+                                    { "url":"www.twitter.com", "title":"Twitter" }]}],
+      "sectionLinks": [{"title": "About us", "links": [{ "url":"www.mystore.com/faq", "title":"FAQ" },
```
Comment (theomoura) [inline README.md]: Could you better indent this part?

File: README.md [hunk 2]
```diff
@@ -49,16 +69,16 @@ Through the Storefront you can change the behavior and interface of the footer.
...
-| `showPaymentFormsInColor` | `Boolean` | Set color of payment forms |
-| `showSocialNetworksInColor` | `Boolean` | Set color of social networks |
-| `logo` | `String` | Link of Store logo |
+| Prop name | Type | Description | Default Value |
+| --- | --- | --- | ------ |
+| `socialNetworks` | `Array(Social Network)` | Array of social networks | [{socialNetwork: 'Facebook', url: '#',},] |
+| `sectionLinks` | `Array(Section Link)` | Array of section links | [] |
+| `storeInformations` | `Array(String)` | Array of informations about the store | [] |
+| `badges` | `Array(Badge)` | Array of badges | [] |
+| `paymentForms` | `Array(Payment Form)` | Array of payment forms | [{title: 'editor.footer.paymentForms.paymentForm',paymentTypes: ['MasterCard'],},] |
```
Comment (theomoura) [inline README.md]: The default section here is hard to understand, consfusing

File: README.md [hunk 3]
```diff
@@ -49,16 +83,16 @@ Through the Storefront you can change the behavior and interface of the footer.
...
+| Prop name | Type | Description | Default Value |
+| --- | --- | --- | ------ |
+| `socialNetworks` | `Array(Social Network)` | Array of social networks | [{socialNetwork: 'Facebook', url: '#',},] |
+| `sectionLinks` | `Array(Section Link)` | Array of section links | [] |
+| `storeInformations` | `Array(String)` | Array of informations about the store | [] |
+| `badges` | `Array(Badge)` | Array of badges | [] |
+| `paymentForms` | `Array(Payment Form)` | Array of payment forms | [{title: `Payment Form/Forma de pago/Forma de pagamento`, paymentTypes: ['MasterCard']}] |
+| `showPaymentFormsInColor` | `Boolean` | Set color of payment forms | false |
+| `showSocialNetworksInColor` | `Boolean` | Set color of social networks | false |
+| `logo` | `String` | Link of Store logo | -
```
Comment (GustavoMA) [inline README.md]:
```suggestion
| `logo` | `String` | Link of Store logo | - |
```

File: README.md [hunk 4]
```diff
@@ -49,16 +83,16 @@ Through the Storefront you can change the behavior and interface of the footer.
...
-| `sectionLinks` | `Array(Section Link)` | Array of section links |
-| `storeInformations` | `Array(String)` | Array of informations about the store |
-| `badges` | `Array(Badge)` | Array of badges |
-| `paymentForms` | `Array(Payment Form)` | Array of payment forms |
-| `showPaymentFormsInColor` | `Boolean` | Set color of payment forms |
-| `showSocialNetworksInColor` | `Boolean` | Set color of social networks |
-| `logo` | `String` | Link of Store logo |
+| Prop name | Type | Description | Default Value |
+| --- | --- | --- | ------ |
+| `socialNetworks` | `Array(Social Network)` | Array of social networks | [{socialNetwork: 'Facebook', url: '#',},] |
```
Comment (GustavoMA) [inline README.md]:
```suggestion
| `socialNetworks` | `Array(Social Network)` | Array of social networks | `[ { socialNetwork: 'Facebook', url: '#' } ]` |
```

Comment (GustavoMA) [reply README.md]: Or

Comment (GustavoMA) [reply README.md]:
```suggestion
| `socialNetworks` | `Array(Social Network)` | Array of social networks | ```[ { socialNetwork: 'Facebook', url: '#' } ]``` |
```

Comment (GustavoMA) [reply README.md]: See which one looks better.

File: README.md [hunk 5]
```diff
@@ -49,16 +83,16 @@ Through the Storefront you can change the behavior and interface of the footer.
...
+| `sectionLinks` | `Array(Section Link)` | Array of section links | [] |
+| `storeInformations` | `Array(String)` | Array of informations about the store | [] |
+| `badges` | `Array(Badge)` | Array of badges | [] |
+| `paymentForms` | `Array(Payment Form)` | Array of payment forms | [{title: `Payment Form/Forma de pago/Forma de pagamento`, paymentTypes: ['MasterCard']}] |
```
Comment (GustavoMA) [inline README.md]: Same thing here.

File: README.md [hunk 6]
```diff
@@ -19,22 +19,56 @@ See our [LTS policy](https://github.com/vtex-apps/awesome-io#lts-policy) for mor
...
     "vtex.store-footer: 2.x"
   }
 ```
 
-Then, add `footer` block into our app theme, like we do in our [Dreamstore app](https://github.com/vtex-apps/dreamstore/blob/master/store/blocks.json).
+Then, add `footer` block into our app theme, like we do in our [Store theme app](https://github.com/vtex-apps/store-theme/blob/master/store/blocks.json). 
+
+Now, you can change the behavior of the footer block. See an example of how to configure: 
+```json
+"footer": {
```
Comment (GustavoMA) [inline README.md]: use 2 space indentation

File: README.md [hunk 7]
```diff
@@ -1,15 +1,15 @@
 # VTEX Footer
 
 ## Description
-The VTEX Footer app shows information about the store such as address, social networks and payment methods. Furthermore, it is possible to add hyperlinks for privacy policy, FAQ, benefits and attendance. This is a VTEX app that is used by Dreamstore product.
+The VTEX Footer app shows information about the store such as address, social networks and payment methods. Furthermore, it is possible to add hyperlinks for privacy policy, FAQ, benefits and attendance. This app is used by store theme.
 
 :loudspeaker: **Disclaimer:** Don't fork this project, use, contribute, or open issue with your feature request.
 
 ## Release schedule
 | Release  | Status              | Initial Release | Maintenance LTS Start | End-of-life | Dreamstore Compatibility
```
Comment (GustavoMA) [inline README.md]:
```suggestion
| Release  | Status              | Initial Release | Maintenance LTS Start | End-of-life | Store Compatibility
```

**Pull request merged**